src/server/v2.0/handler/scanexport.go

```diff
@@ -198,6 +213,9 @@ func (se *scanDataExportAPI) GetScanDataExportExecutionList(ctx context.Context,
 	}
 	execs := make([]*models.ScanDataExportExecution, 0)
 	for _, execution := range executions {
+		if execution.UserName != secContext.GetUsername() {
```

Comment:
If line 210 `ListExecutions` by username, I think that function should guarantee the result so here we don't need to check again?

Reply:
Agreed. This check is redundant. Removed.